includes/class-wpsms-newsletter.php

```diff
@@ -181,9 +182,10 @@ public static function getSubscriberByMobile($number)
         }
 
         $placeholders = implode(', ', $metaValue);
-        $sql          = "SELECT * FROM `{$wpdb->prefix}sms_subscribes` WHERE mobile IN ({$placeholders})";
 
-        $result = $wpdb->get_row($sql);
+        $result = $wpdb->get_row(
+            $wpdb->prepare("SELECT * FROM `{$wpdb->prefix}sms_subscribes` WHERE mobile IN (%s)", $placeholders)
```

Comment:
Let's skip this since it's already prepared in development https://capture.dropbox.com/EjLMd6whtzvNddhW

Also, this method will not work; the output will be `WHERE mobile IN ('1111111, 2222222, 333333')`, which is incorrect. @foadadeli